cmd/boulder-ra/main.go

```diff
@@ -66,6 +66,17 @@ type config struct {
 		// you need to request a new challenge.
 		PendingAuthorizationLifetimeDays int
 
+		// EmailValidationTimeout defines how long the RA will wait trying to
```

Comment:
I was thinking that we would just use the timeout on our DNSResolver, which RA only uses for email lookups.